src/ducks/categorization/services.js

```diff
@@ -186,6 +206,35 @@ const getLocalClassifierOptions = transactionsWithManualCat => {
   }
 }
 
+export const reweightWord = (classifier, category, word, frequencyCount) => {
```

Comment:
Can you explain in the function description why this is necessary and how the reweighting works ? Why is log a good choice here ?

Reply:
I added a doc to the function